decoder_test.go

```diff
@@ -230,3 +231,88 @@ func TestDecoder_Unmarshal_Unmarshaler(t *testing.T) {
 		t.Fatal("Failed to proxy Unmarshaler error to the caller")
 	}
 }
+
+// Win32_BIOS
+type miniBIOS struct {
+	Version             string
+	BIOSVersion         []string
+	BiosCharacteristics []uint16
+}
+
+func TestDecoder_Unmarshal_Slices(t *testing.T) {
+	var bioses []miniBIOS
+	query := "SELECT * FROM Win32_BIOS"
+	err := Query(query, &bioses)
```

Comment:
IHMO, test should not depend on target machine. This one tests unmarshal so it would be much better to create a custom *ole object to unmarshal.

Reply:
Going to leave it in a current state until the first fail from anyone :)

Actually I have no ideas how we could create and utilize fake COM-objects easily enough to use it in tests.